common/SmokeTests/SmokeTest/Program.cs

```diff
@@ -53,6 +53,11 @@
             ProcessType(type);
         }
     }
+    catch (ReflectionTypeLoadException ex) when (ex.Message.Contains("does not have an implementation"))
```

Comment:
I was not able to find an attribute to sniff on the assembly that would let us proactively exclude these.  As much as I hate this magic-string approach, it was the only consistent bit of information that I could find to key on.